test/functional/fixtures/runner/browser-dialogs/native-dialogs-in-cross-domain-iframes-test.js

```diff
@@ -0,0 +1,34 @@
+var expect = require('chai').expect;
```

Comment:
The filename is incorrect (`native-dialogs-in-cross-domain-iframes-test.js`). We don't test iframes here yet. Use the same name as for the `.test.js` file above